gatsby-node.js

```diff
@@ -21,12 +22,26 @@ exports.createPages = async ({ graphql, actions, reporter }) => {
     }
   `);
 
-  if (result.errors) {
+  const wordpressResult = await graphql(`
+    {
+      allWpPost {
+        edges {
+          node {
+            slug
+          }
+        }
+      }
+    }  
+  `);
+
+
+  if (result.errors || wordpressResult.errors) {
     reporter.panicOnBuild("Error while running GraphQL query.");
     return;
   }
 
   const posts = result.data.postsRemark.edges;
+  const wordpressPosts = wordpressResult.data.allWpPost.edges;
```

Comment:
@nibab-boo what will happen if `wordpressResult` is undefined or empty?

Reply:
Thank you for going through this.

I think it should be fine for empty since there is nothing to loop through the WordPress posts.

By undefined, do you mean when there is no response?
I am curious how should we approach that or replicate that in development.